### PR TITLE
add fedorabinary check

### PR DESCRIPTION
**Pull request by yinlinchen**
The workflow is check whether the child node is a FedoraBinary, get description, and added to the triple. The test is passed, but failed in Fedora Repository HTTP API Module...

### CODE REVIEW

File: fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/PropertiesRdfContext.java [hunk 1]
```diff
@@ -66,7 +68,14 @@ public PropertiesRdfContext(final FedoraResource resource,
     private Iterator<Triple> triplesFromProperties(final FedoraResource n)
         throws RepositoryException {
         LOGGER.trace("Creating triples for node: {}", n);
-        final Iterator<Property> allProperties = n.getNode().getProperties();
+        final Iterator<Property> allProperties;
+        if (n instanceof FedoraBinary) {
```
Comment (awoods) [inline fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/PropertiesRdfContext.java]: In the case when the FedoraResource is a FedoraBinary, you should be able to concatenate the properties from both the description and the arg node itself. The end.

Comment (yinlinchen) [inline fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/PropertiesRdfContext.java]: I see, so the logic should be first get properties from n.getNode().getProperties() and then if the node is a binary, append description.getNode().getProperties(). 
Can you tell me how to append allProperties?

Comment (awoods) [inline fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/PropertiesRdfContext.java]: Iterators.concat() should do the trick.
http://guava-libraries.googlecode.com/svn-history/r46/trunk/javadoc/com/google/common/collect/Iterators.html

**Comment by yinlinchen** [comment]
All test passed. FCREPO-1290 fixed.

**Pull request closed without merging**